common/src/verify.rs

```diff
@@ -248,10 +248,10 @@ impl CommitSequenceVerifier {
                     )));
                 }
                 // Verify agenda without transactions
-                if agenda.hash != Agenda::calculate_hash(self.header.height, &[]) {
+                if agenda.hash != Agenda::calculate_hash(&[]) {
                     return Err(Error::InvalidArgument(format!(
                         "invalid agenda hash: expected {}, got {}",
```

Comment:
this comment seems misleading

agenda hash reminds me of `impl ToHash256 for Agenda`

Reply:
fixed.